listenbrainz/db/user_setting.py

```diff
@@ -0,0 +1,125 @@
+import sqlalchemy
+from datetime import datetime, timedelta
+from listenbrainz import db
+from listenbrainz.db.exceptions import DatabaseException
+
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def get_pg_timezone():
+    """ Get list of time zones PostgreSQL supports.
+
+    Returns:
+        list of tuple('zone_name', 'utc_offset')
+    """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT * FROM pg_timezone_names
+            ORDER BY name
+        """))
+        timezones = [(row["name"], row["utc_offset"]) for row in result.fetchall()]
+        timezones = standardize_timezone(timezones)
+        return timezones
+
+
+def get(user_id: int):
+    """Get user setting with the row ID of the user in the DB, or create if there's setting history.
```

Comment:
", or create if there's setting history."

Not sure this still applies since this function doesn't create anything. Maybe just remove?

Reply:
This function is called get, but it acts more like get_timezone. I wonder if we should keep the word get, but return a dict with all of the settings that this function fetches. For starters that would only be: { "timezone": "utc" }, but this allows us to add future user settings to this dict as needed.